src/Styles.res

```diff
@@ -17,7 +17,6 @@ global(.
 global(. "a", [color(hex("303030")), textDecoration(none)])
 global(. "a:hover", [textDecoration(underline)])
 global(. "a:active", [color(hex("303030"))])
-global(. "a:visited", [color(hex("303030"))])
```

Comment:
If you want a `a` tag to show if it has already been visited put that in on a case by case basis. I don't think it should be the global rule